_data/changelogs/docs-accessibility.yml

```diff
@@ -2,7 +2,7 @@ title: Accessibility
 type: documentation
 changelogURL:
 items:
-  - date: NNNN-NN-NN
+  - date: 2024-03-29
```

Comment:
Note: We neglected to update this date when we merged [USWDS-Site - Accessibility page: Replace content #2561](https://github.com/uswds/uswds-site/pull/2561)